programs/uxd/src/instructions/edit_controller.rs

```diff
@@ -25,14 +26,22 @@ pub struct EditController<'info> {
 #[derive(AnchorSerialize, AnchorDeserialize, Clone, Copy)]
 pub struct EditControllerDepositoriesWeightBps {
     pub identity_depository_weight_bps: u16,
-    pub mercurial_vault_depository_0_weight_bps: u16,
-    pub credix_lp_depository_0_weight_bps: u16,
+    pub mercurial_vault_depository_weight_bps: u16,
+    pub credix_lp_depository_weight_bps: u16,
+}
+
+#[derive(AnchorSerialize, AnchorDeserialize, Clone, Copy)]
+pub struct EditControllerDepositories {
```

Comment:
Should we specify somehow here that it's related to `Router`, maybe in the name?

Like that it seems a duplicata of `controller.registered_mercurial_vault_depositories`

Reply:
Yes I think that should appear somewhere indeed

Reply:
Sounds good kings, doing now 👌